src/models/account_model.py

```diff
@@ -0,0 +1,18 @@
+#!/usr/bin/env python3
+"""Model for the account entity."""
+
+from config import db
+from sqlalchemy import func
+
+
+class AccountModel(db.Model):
+    """A model that reprsents a row in the cluster info table."""
+
+    __tablename__ = "account"
+    account_id = db.Column(db.Integer, primary_key=True)
+    last_updated_time = db.Column(db.DateTime)
```

Comment:
Dont we need to map the full table here?

Reply:
This is the table that has just the account data. It only has 3 fields.